assets/src/room_ui.ts

```diff
@@ -57,6 +93,14 @@ export function setupControls(
     callbacks.onLeave();
     window.location.reload();
   };
+
+
+  simulcastControlsBtn.onclick = () => {
+    state.showSimulcast = !state.showSimulcast
+    setShowSimulcast(state.showSimulcast);
+  }
+  simulcastControls.hidden = true
```

Comment:
```suggestion
  simulcastControlsBtnDiv.hidden = true
```
Also, do we have to hide button when we have already hide the whole div?